profiles/coreos/targets/generic/make.defaults

```diff
@@ -33,6 +33,7 @@ INSTALL_MASK="${INSTALL_MASK}
   /etc/sudoers
   /etc/wgetrc
   /etc/xinetd.d
+  /etc/libnl
```

Comment:
This list is alphabetical.  (Or it should be ... `lvm` does not go before `logrotate`.)